frontend/src/pages/ActivityReport/index.js

```diff
@@ -140,14 +140,34 @@ export const findWhatsChanged = (object, base) => {
       return accumulator;
     }
 
+    // this block intends to fix an issue where multi recipients are removed from a report
+    // after goals have been saved we pass up the removed recipients so that their specific links
+    // to the activity report/goals will be severed on the backend
+    if (current === 'activityRecipients' && !isEqual(base[current], object[current])) {
+      // eslint-disable-next-line max-len
+      const grantIds = object.activityRecipients.map((activityRecipient) => activityRecipient.activityRecipientId);
```

Comment:
is there a reason the activityRecipientId's are being put into a variable called grantIds?

Reply:
those numbers, for whatever reason, are the grant ids, not the ids in the sense of activityRecipient.id

Reply:
what value does it have when the activity recipients are other entities? as they don't have grants.